docs/releases/3.1.txt

```diff
@@ -70,8 +70,18 @@ JSONField for all supported database backends
 Django now includes :class:`.models.JSONField` and
 :class:`forms.JSONField <django.forms.JSONField>` that can be used on all
 supported database backends. Both fields support the use of custom JSON
-encoders and decoders. The model field supports the introspection, lookups, and
-transforms that were previously PostgreSQL-only.
+encoders and decoders. The model field supports the :ref:`introspection,
+lookups, and transforms <querying-jsonfield>` that were previously
+PostgreSQL-only::
+
+    from django.db import models
+
+    class ContactInfo(models.Model):
+        data = models.JSONField()
+
+    ContactInfo.objects.create(data={"name": "John", "city": "London"})
+    ContactInfo.objects.filter(data__name="John") \
+        .update(data__city="Cambridge")
```

Comment:
This is not supported until [#29112](https://code.djangoproject.com/ticket/29112) is fixed.